Documentation/Blazorise.Docs/wwwroot/blazorise.docs.js

```diff
@@ -17,6 +17,10 @@
         },
 
         generateToc: (targetElement, options) => {
+            if (!document.getElementById('TableOfContents')) {
```

Comment:
Why do we need this?

Reply:
This is the navigation on the right side

![image](https://github.com/Megabit/Blazorise/assets/900302/4a73bbec-e371-423f-b256-4d8046a26a0b)